nav2_constrained_smoother/include/nav2_constrained_smoother/options.hpp

```diff
@@ -72,7 +72,7 @@ struct SmootherParams
       node, local_name + "w_dist", rclcpp::ParameterValue(0.0));
     node->get_parameter(local_name + "w_dist", distance_weight);
     nav2_util::declare_parameter_if_not_declared(
-      node, local_name + "w_smooth", rclcpp::ParameterValue(15000.0));
+      node, local_name + "w_smooth", rclcpp::ParameterValue(2000000.0));
```

Comment:
Docs update to match?

Reply:
Sure, if you are OK with `w_smooth` 2M new value (and also with this change in general), I'll submit a new doc update PR.

Reply:
Please do

Reply:
Done: https://github.com/ros-planning/navigation.ros.org/pull/385. Seems to be mentioned only on two places, there is no tutorials or something else needed to be updated.